easy_my_coop/migrations/12.0.3.1.0/pre-migration.py

```diff
@@ -0,0 +1,3 @@
+def migrate(cr, version):
+    cr.execute("UPDATE subscription_request SET firstname='-' WHERE firstname IS NULL;")
+    cr.execute("UPDATE subscription_request SET lastname='-' WHERE lastname IS NULL;")
```

Comment:
I had a look at the data on a few production databases (see below), these values seem reasonable since for most forms these fields have been correctly set. We won't end up with most records named "-" (cf `_rec_name`)

```sql
odoo12@(none):bees>
 select firstname is null as null_firstname,
        lastname is null  as null_lastname,
        count(0)
 from subscription_request
 group by 1, 2
+------------------+-----------------+---------+
| null_firstname   | null_lastname   | count   |
|------------------+-----------------+---------|
| False            | False           | 4653    |
| True             | True            | 21      |
| False            | True            | 2       |
| True             | False           | 2       |
+------------------+-----------------+---------+
SELECT 4
Time: 0.015s
---
odoo12@(none):lenid>
.................... select firstname is null as null_firstname,
....................        lastname is null  as null_lastname,
....................        count(0)
.................... from subscription_request
.................... group by 1, 2
+------------------+-----------------+---------+
| null_firstname   | null_lastname   | count   |
|------------------+-----------------+---------|
| True             | True            | 3       |
| False            | False           | 125     |
| False            | True            | 756     |
+------------------+-----------------+---------+
SELECT 3
Time: 0.043s
---
odoo12@brasseriedelorne>
................. select firstname is null as null_firstname,
.................        lastname is null  as null_lastname,
.................        count(0)
................. from subscription_request
................. group by 1, 2
+------------------+-----------------+---------+
| null_firstname   | null_lastname   | count   |
|------------------+-----------------+---------|
| True             | True            | 3       |
| False            | False           | 402     |
+------------------+-----------------+---------+
SELECT 2
Time: 0.089s
```